### PR TITLE
EditText Minor update

### PR DESCRIPTION
**Pull request by DevYuns**
## Description

 Layout fix, text change

# Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: main/EditText/index.tsx [hunk 1]
```diff
@@ -198,15 +199,11 @@ function EditText(props: Props): ReactElement {
             autoCapitalize={autoCapitalize}
             onFocus={(): void => {
               setFocus(true);
-              if (onFocus) {
-                onFocus();
-              }
+              onFocus && onFocus();
```
Comment (JeffGuKang) [inline main/EditText/index.tsx:202]: How about using optional chaining? 
```
onFocus?.();
```

File: main/EditText/index.tsx [hunk 2]
```diff
@@ -198,15 +199,11 @@ function EditText(props: Props): ReactElement {
...
             }}
             onBlur={(): void => {
               setFocus(false);
-              if (onBlur) {
-                onBlur();
-              }
+              onBlur && onBlur();
```
Comment (JeffGuKang) [inline main/EditText/index.tsx:206]:
```
onBlur?.()
```

File: main/EditText/index.tsx [hunk 3]
```diff
@@ -327,83 +322,69 @@ function EditText(props: Props): ReactElement {
...
+                setFocus(true);
+                if (onFocus) {
+                  onFocus();
+                }
+              }}
+              onBlur={(): void => {
+                setFocus(false);
+                if (onBlur) {
+                  onBlur();
+                }
```
Comment (JeffGuKang) [inline main/EditText/index.tsx:357]:
```
onBlur?.();
```

File: main/EditText/index.tsx [hunk 4]
```diff
@@ -327,83 +322,69 @@ function EditText(props: Props): ReactElement {
...
+          >
+            <StyledTextInput
+              {...textInputProps}
+              testID={testID}
+              autoCapitalize={autoCapitalize}
+              onFocus={(): void => {
+                setFocus(true);
+                if (onFocus) {
+                  onFocus();
+                }
```
Comment (JeffGuKang) [inline main/EditText/index.tsx:351]: Same with above. 
```
onFocus?.();
```

File: main/EditText/index.tsx [hunk 5]
```diff
@@ -327,83 +322,69 @@ function EditText(props: Props): ReactElement {
...
-                  !leftElement
-                    ? { paddingLeft: 15 }
-                    : null,
-                  !rightElement
-                    ? { paddingRight: 15 }
-                    : null,
-                ]}
-              />
-              : null
-          }
```
Comment (JeffGuKang) [inline main/EditText/index.tsx]: Don't we need this anymore? Just I am curious about it.

File: main/EditText/index.tsx [hunk 6]
```diff
@@ -468,13 +447,11 @@ function EditText(props: Props): ReactElement {
...
             />
           </StyledRowContent>
-          {
-            errorText
-              ? <StyledInvalidText testID={errorTestID} style={errorTextStyle}>
-                {errorText}
-              </StyledInvalidText>
-              : null
-          }
+          {errorText ? (
```
Comment (JeffGuKang) [inline main/EditText/index.tsx]: It can be `{errorText && ...}` as you already did.

File: main/EditText/index.tsx [hunk 7]
```diff
@@ -327,83 +322,69 @@ function EditText(props: Props): ReactElement {
...
+              multiline={multiline}
+            />
+            {rightElement && (
+              <StyledIcon style={[{ width: 40 }, rightElementStyle]}>
+                {rightElement}
+              </StyledIcon>
+            )}
+
+          </StyledRowContent>
+          {errorText ? (
```
Comment (JeffGuKang) [inline main/EditText/index.tsx:375]: It can be `&&` like 368 line

File: main/EditText/index.tsx (hunk 1 above)
Comment (hyochan) [reply main/EditText/index.tsx:202]: Good idea. However, we need to [upgrade the typescript](https://github.com/dooboolab/dooboo-ui/blob/fa61f04b7789236ceb8d3957f9da82541d430dc3/package.json#L63) beforehand.

File: main/EditText/index.tsx [hunk 8]
```diff
@@ -37,6 +37,7 @@ const StyledRowInput = styled.TextInput`
   flex: 1;
   height: 100%;
   color: #2c374e;
+  ${Platform.OS === 'web' && { 'outline-style': 'none' }}
```
Comment (hyochan) [inline main/EditText/index.tsx:40]: @DevYuns I think you can tell the effect of this in more detail. Maybe some screenshots.

Comment (DevYuns) [reply main/EditText/index.tsx:40]: TextInput element already has border-width as you can see in line 397. But when clicking the TextInput element(I mean when element is focused), It show two line overlapped. one is focus outline, the other is border-width in web(The focus outline doesn't appear in mobile). So I add this code. It make focus-outline invisible in web. :)

Comment (DevYuns) [reply main/EditText/index.tsx:40]: Screen Shot 2020-06-17 at 9 53 32

File: main/EditText/index.tsx (hunk 6 above)
Comment (DevYuns) [reply main/EditText/index.tsx]: Thank you for your reviewing. When I try to change that code to '&& syntax', It make an error.   I can't understand why that happened. Could you check this error? Thank you in advance. :)


Screen Shot 2020-06-17 at 10 03 38

Screen Shot 2020-06-17 at 10 02 53

File: main/EditText/index.tsx (hunk 5 above)
Comment (DevYuns) [reply main/EditText/index.tsx]: With this code, TextInput layout is little bit twisted in web.

Comment (DevYuns) [reply main/EditText/index.tsx]: When I remove this code, It does't make any side effect in my sight.

**Review by hyochan — APPROVED** [review]
LGTM

File: main/EditText/index.tsx (hunk 1 above)
Comment (JeffGuKang) [reply main/EditText/index.tsx:202]: I got it. Let's leave it for the next.

**Review by JeffGuKang — APPROVED** [review]
Looks nice.

**Pull request merged**